proto/regen/ecocredit/v1alpha1/types.proto

```diff
@@ -73,10 +76,20 @@ message Params {
   // credit_class_fee is the fixed fee charged on creation of a new credit class
   repeated cosmos.base.v1beta1.Coin credit_class_fee = 1
       [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"];
+
+  // credit_types is a list of definitions for credit types
+  repeated CreditType credit_types = 2;
 }
 
 // GenesisState defines the state of the ecocredit module that is needed at genesis
 message GenesisState {
   // Params contains the updateable global parameters for use with the x/params module
   Params params = 1 [(gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"params\""];
+}
+
+// CreditType defines the measurement unit/precision of a certain credit type (e.g. carbon, biodiversity...)
+message CreditType {
+  string type = 1;
+  string unit = 2;
```

Comment:
Let's add more docs, i.e. `type` should be something like `carbon` and `units` should be some SI unit like `tons`.
```suggestion
  string type = 1;
  string units = 2;
```

Reply:
i ninja'd some docs in right before this review comment 😏 

also changed unit -> units 👍🏻

Reply:
`type` may cause problems in some langauges, where `type` is a keyword. Eg: typescript, haskell...

Reply:
@technicallyty did you address Robert's comment above?